rainier-core/src/main/scala/com/stripe/rainier/compute/Translator.scala

```diff
@@ -96,6 +96,28 @@ private class Translator {
   The result may also be multiplied by a constant scaling factor (generally
   factored out of the original summation).
   **/
+  private def factoredSumLine(ax: Coefficients,
+                              b: BigDecimal,
+                              factor: Double): Expr = {
+    val ring = multiplyRing
```

Comment:
is there any point to using `ring` here instead of just inlining all of the ops/values below? It seems like unnecessary indirection.

Reply:
Inlining is better. Fixed.